src/main/java/bro.java

```diff
@@ -0,0 +1,215 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.*;
+
+public class bro {
+
+    private static final ArrayList<Task> storer = new ArrayList<>();
+    private static final String FILE_PATH = "data/duke.txt";
+
+    public static void level0() {
+        System.out.println("Hello! I'm bro");
+        System.out.println("What can I do for you?");
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public static void saveTasks() {
+        try {
+            File file = new File(FILE_PATH);
+            file.getParentFile().mkdirs();
+            FileWriter writer = new FileWriter(file);
+
+            for (Task task : storer) {
+                writer.write(task.toSave() + System.lineSeparator());
+            }
+            writer.close();
+        } catch (IOException e) {
+            System.out.println("Error while saving tasks: " + e.getMessage());
+        }
+    }
+
+    public static void loadTasks() {
+        File file = new File(FILE_PATH);
+        if (!file.exists()) {
+            System.out.println("No previous tasks found. Please create a new task list.");
+            return;
+        }
+
+        try (Scanner scanner = new Scanner(file)) {
+            while (scanner.hasNext()) {
+                String line = scanner.nextLine();
+                String[] parts = line.split(" \\| ");
+                switch(parts[0]) {
+                    case "T":
+                        Todo todo = new Todo(parts[2]);
+                        if (parts[1].equals("1")) {
+                            todo.markAsDone();
+                        }
+                        storer.add(todo);
+                        break;
+                    case "D":
+                        Deadline ddl = new Deadline(parts[2], parts[3]);
```

Comment:
Variables like ddl, taskInt, and task_num can be more descriptive. For instance, use deadlineTask instead of ddl, and taskIndex instead of taskInt. Consistency in naming conventions (camelCase) is also important